dataset.proto

```diff
@@ -9,9 +9,11 @@ import "zvelo.io/msg/category.proto";
 enum DataSetType {
   CATEGORIZATION = 0;
   ADFRAUD        = 1;
-  // 2 and 3 are reserved
+  // 2 is reserved
```

Comment:
Not related to this PR, but I'd like to know why we need to reserve proto indices? At this point it feels a bit premature.